lib/src/throughput/EwmaEstimator.ts

```diff
@@ -0,0 +1,60 @@
+import type { ResourceTiming } from '../request/ResourceTiming'
+import { Ewma } from './Ewma';
+import type { ThroughputEstimator } from './ThroughputEstimator'
```

Comment:
Please add `.ts` file extensions to all imports

Reply:
I wonder if I miss any extension of config, as by default I don't have *.ts added to all imports?

Reply:
Addressed it in https://github.com/streaming-video-technology-alliance/common-media-library/pull/160/commits/d0834e6fe6715ff4169b03ae6dd810bce82c80e7

Reply:
Clarified that the extensions must be `.js`, not `.ts`. I addressed it in https://github.com/streaming-video-technology-alliance/common-media-library/pull/160/commits/0a2c924b1adb511ff0e4246275f24ffc668b15cf